### DISCUSSION TITLE
admin切换用户以后无法登录

### DISCUSSION THREAD

**Discussion by hackallan** [apache/streampark#341] [General]
用admin登录以后，创建了新用户，然后用新用户登录时候一直处于白板状态，补充：启动console以后需要启动webapp服务吗？webapp服务的目的是什么？
![image](https://user-images.githubusercontent.com/18672845/136539833-a6f62e85-68c4-418f-9192-4939a53d1323.png)

**Comment by wolfboys** [comment]
F12看看浏览器控制台和网络请求.

**Reply by hackallan** [reply]
这个问题，其实是因为添加的用户是develop权限，而develop默认没有勾选application的权限，所以只能是白板。所以一开始就要去设置一些角色可以看到哪些页面。但是即时这样我觉得应该最起码的右上角的那些注销登录的信息还是应该有的，要不就一直卡在这里了，除非清空一下缓存，然后重新登录
